Wikipedia/Code/ShareActivityController.swift

```diff
@@ -0,0 +1,94 @@
+@objc(WMFShareActivityController)
+class ShareActivityController: UIActivityViewController {
+    
+    init(url: URL, userDataStore: MWKDataStore, context: AnalyticsContextProviding) {
```

Comment:
`url:` should be renamed `articleURL` to make it clear it's expecting an `articleURL`